### ISSUE TITLE
Carriage return + blank line after the open brace

### ISSUE THREAD

**Issue by yoanmalie** [KVXYMatt/class-extractor#9]
Hello,
Firstly, thanks for this extension it's so helpful !

It would be nice to put a carriage return after the open brace `{`, and a blank line. So instead having this:
```
.c-thumb__cover { }
.c-thumb__header { }
.c-thumb__content { }
```

We got:

```
.c-thumb__cover {

}
.c-thumb__header {

}
.c-thumb__content {

}
```

In that way we're ready to put some styles here.

Thanks!

**Comment by KVXYMatt** [comment]
Hi!

Thanks for your great idea. I've added a setting in the extension to let you pick the output format of your choice.

**Comment by yoanmalie** [comment]
wow thanks for the quick add! 
How can I try it? Because I don't see the update on VS.

**Comment by KVXYMatt** [comment]
Sorry I'm just trying to clean up the BEM formatting issues since they're related. I'm hoping that'll be done in the next day or so and I'll push a new update when I can

**Comment by KVXYMatt** [comment]
Sorry for the delay, this has been updated and released in 1.1.0 just published to the marketplace. Feel free to update the setting now available, but just take note that VS Code doesn't render the line break character and it escapes backslashes if you edit the config option in the visual settings editor, so use the JSON settings editor for this. Any other issues let me know
